VMEncryption/main/handle.py

```diff
@@ -932,7 +931,7 @@ def encrypt_inplace_without_seperate_header_file(passphrase_file,
             logger.log(msg="the current phase is " + str(CommonVariables.EncryptionPhaseBackupHeader),
                        level=CommonVariables.InfoLevel)
 
-            if not ongoing_item_config.get_file_system().lower() in ["ext2", "ext3", "ext4"]:
+            if not ongoing_item_config.get_file_system().lower() in CommonVariables.inplace_supported_file_systems:
                 logger.log(msg="we only support ext file systems for centos 6.5/6.6/6.7 and redhat 6.7",
```

Comment:
while in this area can we also update the log message issued in the next line to be more generic and not name specific distros and versions to be more future proof (this has led to support questions since some of those are no longer on our support list) - see same comment made on the single pass PR